tt_metal/hw/inc/debug/sanitize_noc.h

```diff
@@ -129,19 +129,17 @@ uint32_t debug_sanitize_noc_addr(
 
     // Check noc addr, we save the alignment requirement from the noc src/dst because the L1 address
     // needs to match alignment.
-    uint32_t alignment_mask = NOC_L1_ALIGNMENT_BYTES - 1;  // Default alignment, only override in ceratin cases.
+    uint32_t alignment_mask = (dir == DEBUG_SANITIZE_NOC_READ ? NOC_L1_READ_ALIGNMENT_BYTES : NOC_L1_WRITE_ALIGNMENT_BYTES) - 1;  // Default alignment, only override in ceratin cases.
```

Comment:
@tt-dma can you review these changes?

Reply:
Looks good to me, can we just update the comments on line 135/141? They're out of date now that we have restrictions for writes as well